pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/SkylineBatchConfigForm.cs

```diff
@@ -52,7 +52,7 @@ public SkylineBatchConfigForm(IMainUiControl mainControl, SkylineBatchConfig con
             _newReportList = new List<ReportInfo>();
 
             _mainControl = mainControl;
-            _initialEnabled = config.Enabled;
+            _initialEnabled = config?.Enabled ?? false;

```

Comment:
Could also just use

if (config != null)
    _initialEnabled = config.Enabled

Because the default values is always false. Not quite as fancy on the syntax and requiring 2 lines, but to me simpler to read.